### ISSUE TITLE
MS.CA.Common has a broken analyzer dependency

### ISSUE THREAD

**Issue by jaredpar** [dotnet/roslyn#4774]
The Microsoft.CodeAnalysis.Common NuGet package has a dependency on Microsoft.CodeAnalysis.Analyzers.  This dependency assumes the version number of the packages will always be in sync.  

This version dependency was fine up till VS 2015 RTM as both NuGets were produced from the same repo in the same solution.  Now the NuGets are split up into separate repos and produced an unrelated times.  This needs to be fixed by either:

1. Removing the dependency on MS.CA.Analyzers. 
2. Change the dependency to be a fixed version, possibly 1.0.0 RTM

**Comment by tmeschter** [comment]
Another option is to make Microsoft.CodeAnalysis.Common depend on Microsoft.CodeAnalysis.Analyzers 1.0.0 or above, rather than *only* 1.0.0.

**Comment by svick** [comment]
Is this the cause of #4741?

**Comment by jaredpar** [comment]
@svick yes I think that is the case.

**Comment by genlu** [comment]
@jaredpar How can I verify if this is fixed or not?

**Comment by jaredpar** [comment]
@genlu install the package on a machine and verify that it installs :smile:

**Comment by genlu** [comment]
@jaredpar OK, let me rephrase my question :):
1. How do I build the packages locally? Or,
2. How can I access the Jenkins build drop if the package build is included?
3. Where are the built packages located?

**Comment by jaredpar** [comment]
@genlu build Roslyn.sln and then Setup.proj.  That will build the NuGet.

**Comment by genlu** [comment]
@jaredpar The fix is verified, please see the screenshot:

![snip_20150911110753](https://cloud.githubusercontent.com/assets/788783/9822454/6b423342-5875-11e5-9b5a-97f9be359f02.png)

However, I encountered an error when built `setup.proj` (more specifically,  `Open\src\Dependencies\Microsoft.DiaSymReader\Microsoft.DiaSymReader.nuget.proj`): the file `NetFX20\Microsoft.DiaSymReader.dll` was not found. I have to manually build `\Open\src\Dependencies\Microsoft.DiaSymReader\NetFX2\Microsoft.DiaSymReader.NetFX20.csproj` to proceed. Is this a known issue? Or did I do anything wrong?

**Comment by jaredpar** [comment]
@tmat can answer that.  It should work IMHO

**Comment by tmat** [comment]
You need to build Dependencies\Dependencies.sln.

**Comment by genlu** [comment]
Fixed by https://github.com/dotnet/roslyn-internal/issues/104 and https://github.com/dotnet/roslyn/issues/4783
